OtripleS.Portal.Web/Models/Users/User.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using Microsoft.AspNetCore.Identity;
+using System;
+
+namespace OtripleS.Portal.Web.Models.Users
+{
+    public class User : IdentityUser<Guid>
+    {
+        public override Guid Id
+        {
+            get => base.Id;
+            set => base.Id = value;
+        }
+
+        public override string UserName
+        {
+            get => base.Email;
+            set => base.Email = value;
+        }
+
+        public override string PhoneNumber
+        {
+            get => base.PhoneNumber;
+            set => base.PhoneNumber = value;
+        }
+
+        public string Name { get; set; }
+        public string FamilyName { get; set; }
+        public UserStatus Status { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
```

Comment:
I don't think you need that here. At this point a a consumer you just want to have the properties without the whole inheritance and override action.

Reply:
Fixed it.